tests/component/.metadata/nuget.config

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+
+    <!--
+      Temporary private nuget feed to serve up winmdgenerator 0.3.2-preview.
```

Comment:
I thought Steve fixed the nuget feed?

Reply:
Will pull down the new winmdgenerator and update the PR.